tests/backends/test_dbmodel.py

```diff
@@ -17,6 +17,8 @@
 
 should_run &= "tensorflow" in installed_redisai_backends()
 
+if not should_run:
```

Comment:
Could be worth removing the rest of the skip checks after this, since they're made redundant by `allow_module_level=True`

Reply:
Good catch, done!

Reply:
Good catch, done!